easybuild/easyconfigs/r/REMORA/REMORA-1.8.2-goolf-1.7.20.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'PackedBinary'
+
+name = "REMORA"
+version = "1.8.2"
+
+homepage = 'https://github.com/TACC/remora'
+description = """REsource MOnitoring for Remote Applications"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/TACC/remora/archive/']
+sources = ['v%(version)s.tar.gz']
+
+unpack_options = '--strip-components=1'
+
+install_cmd = 'REMORA_INSTALL_PREFIX=%(installdir)s ./install.sh'
+
+modextravars = {
+    'REMORA_BIN': '%(installdir)s/bin',
+}
+
+sanity_check_paths = {
+    'files': ['bin/remora'],
```

Comment:
also include `'bin/mpstat'`, see the hard `cp mpstat $REMORA_DIR/bin` in `install.sh`

also, when installing on top of an MPI that is not Intel MPI, you should also have `libmpiP.*` installed, so:

```python
    'files': ['bin/mpstat', 'bin/remora', 'lib/libmpiP.a', 'lib/libmpiP.%s' % SHLIB_EXT],
```